### PR TITLE
Add YARD doc links between README.md and MemoWise

### PR DESCRIPTION
**Pull request by ms-ati**

### CODE REVIEW

**Comment by ms-ati** [comment]
See #34

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 ## Why MemoWise?
 
-**MemoWise is the wise choice for memoization in Ruby.**
+**{MemoWise} is the wise choice for memoization in Ruby.**
```
Comment (JacobEvelyn) [inline README.md:11]: I'm not sure I understand this... won't this look weird in the GitHub `README.md` view?

Comment (ms-ati) [reply README.md:11]: Ah sure -- I can move the link elsewhere

File: README.md [hunk 2]
```diff
@@ -33,6 +33,9 @@ Or install it yourself as:
 
 ## Usage
 
+See API docs: {MemoWise} (hosted
+[here](http://rubydoc.info/github/panorama-ed/memo_wise)).
```
Comment (ms-ati) [inline README.md:37]:
```suggestion
See [API docs](https://rubydoc.info/github/panorama-ed/memo_wise/MemoWise) for more details.
```

**Review by JacobEvelyn — APPROVED** [review]

**Pull request merged**